alfresco-transformer-base/src/main/java/org/alfresco/transformer/logging/StandardMessages.java

```diff
@@ -28,10 +28,7 @@
 
 public interface StandardMessages
 {
-    String ENTERPRISE_LICENCE = 
-        "This image is only intended to be used with the Alfresco Enterprise Content Repository which is covered by\n"+
-        "https://www.alfresco.com/legal/agreements and https://www.alfresco.com/terms-use\n" +
-        "\n" +
+    String LICENCE =
         "License rights for this program may be obtained from Alfresco Software, Ltd. pursuant to a written agreement\n" +
         "and any use of this program without such an agreement is prohibited.\n" +
         "\n" ;
```

Comment:
We may wish to change the text of the **`LICENCE`** constant to reference _LGPLv3_?
@montgolfiere

Reply:
Yes, can we find any example log startup text from other core ACS Community components - Repo, Search, Share, ... ?

Reply:
Can't find anything similar to this around the mentioned repos so I'll proceed merging.
If we'll manage to find something we can easily update the `String`.